packages/superset-ui-legacy-preset-chart-nvd3/src/NVD3Vis.css

```diff
@@ -121,3 +121,13 @@
 .superset-legacy-chart-nvd3-tr-total {
   font-weight: bold;
 }
+
+/* START tooltip styles - these prevent long column names from overflowing the tooltip */
+.nvtooltip tbody tr:first-child td{
+  white-space: nowrap;
+  font-weight: bold;
+}
+.nvtooltip tbody tr:not(:first-child) td:nth-child(2){
+  word-break: break-word;
+}
+/* END tooltip styles */
```

Comment:
nit: add a new line

Reply:
Committed. I wish people didn't object to using Husky to add pre-commit hooks running Prettier

Reply:
who objects to husky? I support that 👍

Reply:
on second thought, I'm not so sure. we've had these in other repos and  the major downside of pre-commit hooks is that they can take longer and longer to run, especially as the repo grows. 

I think you can setup your `ide` to auto-fix eslint (which calls prettier) changes on save.